ansible_collections/arista/cvp/docs/modules/cv_validate_v3.rst.md

```diff
@@ -0,0 +1,98 @@
+# cv_validate_v3
+
+Online/Offline configlet validation
+
+Module added in version 3.7.0
```

Comment:
```suggestion
Module added in version 3.7.0.
```

Reply:
the `.` after the versioning looks very strange to me! I would keep it as is `3.7.0`